migrations/postgresql/2024-01-12-210182_change_attachment_size/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE attachments MODIFY file_size BIGINT NOT NULL;
```

Comment:
This doesn't work for PostgreSQL.

```suggestion
ALTER TABLE attachments
ALTER COLUMN file_size TYPE BIGINT,
ALTER COLUMN file_size SET NOT NULL;
```